service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceLocators.scala

```diff
@@ -102,33 +100,33 @@ trait CircuitBreakerComponents {
  * Components for using the configuration service locator.
  */
 trait ConfigurationServiceLocatorComponents extends CircuitBreakerComponents {
-  lazy val serviceLocator: ServiceLocator = new ConfigurationServiceLocator(configuration, circuitBreakersPanel)(executionContext)
+  lazy val serviceLocator: ServiceLocator = new ConfigurationServiceLocator(config, circuitBreakersPanel)(executionContext)
 }
 
 /**
  * A service locator that uses static configuration.
  */
-class ConfigurationServiceLocator(configuration: Config, circuitBreakers: CircuitBreakers)(implicit ec: ExecutionContext)
+class ConfigurationServiceLocator(config: Config, circuitBreakers: CircuitBreakersPanel)(implicit ec: ExecutionContext)
   extends CircuitBreakingServiceLocator(circuitBreakers) {
 
-  @deprecated(message = "prefer constructor using typesafe Config instead", since = "1.4.0")
+  @deprecated(message = "Use constructor accepting {@link com.lightbend.lagom.scaladsl.client.CircuitBreakersPanel} instead", since = "1.4.0")
```

Comment:
Putting a `@link` tag inside a deprecation annotation like this doesn't render correctly. I think this was a copy/paste from a Javadoc comment.

I don't know what kind of conventions there are for deprecation messages in Scala, but I think putting the full signature in there would be good, like "Use ConfigurationServiceLocator(config: Config, circuitBreakers: CircuitBreakersPanel)(implicit ec: ExecutionContext) instead". Or do square-bracket links work here?